### PR TITLE
Document and implement specification changes

### PR DESCRIPTION
**Pull request by jsancio**

### CODE REVIEW

**Comment by tamarrow** [comment]
@jsancio looks good. should we also update the package validation with the new schema?

**Comment by jsancio** [comment]
@tamarrow Done. I changed the package-schema.json to only allow packages with version `"2.0"`.

File: repo/meta/schema/package-schema.json [hunk 1]
```diff
@@ -68,6 +72,7 @@
     }
   },
```
Comment (tamarrow) [inline repo/meta/schema/package-schema.json]: should we also remove `images` from this schema since they should now be in `resource.json`?

**Comment by tamarrow** [comment]
@jsancio thanks. can we also add a schema for `resource.json` in `repo/meta/schema/resource-schema.json`?

File: repo/meta/schema/package-schema.json (hunk 1 above)
Comment (jsancio) [inline repo/meta/schema/package-schema.json]: Yes. We already removed it. It is part of the version-2 branch. This PR is against version-2. I am fixing the documentation and adding a `packagingVersion` property.

**Comment by jsancio** [comment]
> @jsancio thanks. can we also add a schema for resource.json in repo/meta/schema/resource-schema.json?

We did. It is already part of version-2. This is a PR against that branch.

**Pull request merged**